src/lib/FieldType/Mapper/UserAccountFieldValueFormMapper.php

```diff
@@ -46,22 +46,23 @@ public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
         $formConfig = $fieldForm->getConfig();
         $rootForm = $fieldForm->getRoot()->getRoot();
         $formIntent = $rootForm->getConfig()->getOption('intent');
-        $isTranslation = $rootForm->getData() instanceof ContentTranslationData;
+        $formData = $rootForm->getData();
+        $isTranslation = $formData instanceof ContentUpdateData;
+        if ($isTranslation) {
+            return;
+        }
```

Comment:
Why are you now verifying instance of `ContentUpdateData` instead? If it's needed, at least `$isTranslation` should be properly renamed. Besides, since this variable is used only once, maybe:
```suggestion
        if ($formData instanceof ContentUpdateData) {
            return;
        }
```

Reply:
Because ContentUpdateData is used when translating

Reply:
Isnt that the root of the bug? That `ContentUpdateData` is used for User instead of `ContentTranslationData`?

Reply:
@ViniTou @konradoboza 
translating action goes through `proxyTranslateAction` and then, its response is always `content/edit/draft`, then hits `editVersionDraftAction` and then we are in `ContentEditViewFilter` when we are using `ContentUpdateMapper` -> `ContentUpdateData`. The same happens for every other content which seems logical because we are in fact editing drafts when translating.

Besides even in `TranslationFormProcessor` `ContentUpdateData` is built.

Reply:
Thanks for the clarification. Not sure if we want to block translating users from the business point of view though. Besides, obtaining content type via `$rootForm->getData()->contentDraft->contentType` seems to work.

Reply:
@konradoboza why would anyone want to translate e-mails, usernames, and passwords from the business point of view? Because that's what is included in the user account.

Reply:
> because we are in fact editing drafts when translating.

@barw4 
That's true, and that's why `ContentTranslationData` extends `ContentUpdateStruct`, but this is not the reason why `ContentUpdateData` should be used instead of it as clearly there was different intent when `ContentTranslationData` was introduced.

I agree tho, that `ezuser` ft should not be active when translating.

Reply:
Besides, passing `$contentType` redirects us to the form where the mentioned by @ViniTou fields are disabled anyway.

Reply:
They are not really disabled, they are set to read-only and the whole JS validation pops in.

Reply:
Also, wait a minute, atm you are removing those forms from view (or not adding them to be preciese) only basing on data being `ContentUpdateData`?
So how this behaves when you are editing user?

Reply:
@ViniTou you are getting `UserUpdateData`, editing works properly.

Reply:
But you get `ContentTranslationData` when translating content, right?

Reply:
Have you checked whether editing newly created translation works? Besides, in the form we are getting an empty section:
```
User account
This Field Type is not editable
```
it should not be there if it's empty.

Reply:
Not at all.

Creating - `userCreateData` 
Updating - `userUpdateData`
Translating - `contentUpdateData`

Reply:
> it should not be there if it's empty.

Isn't it actually a good idea to leave it here? This just indicates that this field is "special".

Reply:
Then once again, I would say that not using `ContentTranslationData` when translating content is the real bug here.

Reply:
Okay, but `ContentTranslationData` is not used when translating any content, not just *user* one.

Reply:
Yes, and it just means that the issue is bigger.

I believe similar logic is (disabling when translating) is done with CustomerGroup FT in product catalog. Do we have similar issue there as well?? @adamwojs @Steveb-p

Reply:
~Customer Group is rather a separate object not a Field Type so the approach is quite different there.~ brainfart 😓

Reply:
I'll put this topic on hold until we decide how it should really work and by that I mean:
- usage of `ContentTranslationData `
- UI behavior (should we hide the form? should we show it as disabled? - JS validation has to be fixed in this case)
- external storage behavior - should the `user_account` data be copied? set to null?

Reply:
As I said, it is the same issue here then:
https://github.com/ibexa/product-catalog/blob/main/src/bundle/FieldType/CustomerGroup/FieldValueFormMapper.php#L23

I would say, we should use `ContentTranslationData/UserTranslationData` if needed.
And show form as disabled, as we did till now.

Even in the PR title you stated disabling and not hiding/removing ;)